viscm/cli.py

```diff
@@ -0,0 +1,115 @@
+import sys
+
+import matplotlib.pyplot as plt
+
+from viscm import gui
+
+
+def cli():
+    import argparse
+    argv = sys.argv[1:]
+
+    # Usage:
+    #   python -m viscm
+    #   python -m viscm edit
+    #   python -m viscm edit <file.py>
+    #      (file.py must define some appropriate globals)
+    #   python -m viscm view <file.py>
+    #      (file.py must define a global named "test_cm")
+    #   python -m viscm view "matplotlib builtin colormap"
+    #   python -m viscm view --save=foo.png ...
```

Comment:
This may get out of sync quickly; worth keeping?

Reply:
Good call, we have `--help` for this! And the `README` also contains a couple examples. Though looks like this part is not documented in `--help`:

```
    #   python -m viscm view <file.py>
    #      (file.py must define a global named "test_cm")
```

Reply:
16b067a